clang/test/CodeGen/RISCV/attr-hw-shadow-stack.c

```diff
@@ -0,0 +1,7 @@
+// RUN: %clang_cc1 -triple riscv64-linux-unknown -target-feature +zimop -emit-llvm -o - %s -fcf-protection=return | FileCheck %s
+// RUN: %clang_cc1 -triple riscv64-linux-unknown -target-feature +zimop -emit-llvm -o - %s | FileCheck -check-prefix=NOSHADOWSTACK %s
```

Comment:
The triple components are in the wrong order. They should be `riscv{64,32}-unknown-linux-gnu`.

> Configuration names are strings in the canonical form: ARCHITECTURE-VENDOR-OPERATING_SYSTEM or ARCHITECTURE-VENDOR-OPERATING_SYSTEM-ENVIRONMENT

(ref. https://llvm.org/doxygen/classllvm_1_1Triple.html#details)